src/containers/NonBitGoRecoveryCoin/AlgorandForm.tsx

```diff
@@ -0,0 +1,198 @@
+  fee: Yup.number().required(),
+  firstRound: Yup.number().optional()
+    .typeError('firstRound must be a number')
+    .integer()
+    .positive('firstRound must be a positive integer')
+    .required(),
+  note: Yup.string().optional(),
+  nodeParams: Yup.object({
+    token: Yup.string().required(),
+    baseServer: Yup.string().required(),
+    port: Yup.number().required(),
+  }),
+}).required();
+
+export type AlgorandFormProps = {
+  onSubmit: (
+    values: AlgorandFormValues,
+    formikHelpers: FormikHelpers<AlgorandFormValues>
+  ) => void | Promise<void>;
+};
+
+type AlgorandFormValues = Yup.Asserts<typeof validationSchema>;
+
+export function AlgorandForm({ onSubmit }: AlgorandFormProps) {
+  const formik = useFormik<AlgorandFormValues>({
+    onSubmit,
+    initialValues: {
+      backupKey: '',
+      userKey: '',
+      rootAddress: '',
+      recoveryDestination: '',
+      bitgoKey: '',
+      walletPassphrase: '',
+      fee: 1000,
+      firstRound: 0,
+      note: '',
+      nodeParams: {
+        token: '',
+        baseServer: '',
+        port: 8443,
+      }
+    },
+    validationSchema,
+  });
+
+  return (
+    <FormikProvider value={formik}>
+      <Form>
+        <h4 className="tw-text-body tw-font-semibold tw-border-b-0.5 tw-border-solid tw-border-gray-700 tw-mb-4">
+          Self-Managed Hot Wallet
+        </h4>
+        <div className="tw-mb-4">
+          <FormikTextarea
+            HelperText="Your encrypted user key (box A of the KeyCard)."
+            Label="Encrypted User Key"
+            name="userKey"
+            rows={4}
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextarea
+            HelperText="Your encrypted backup key (box B of the KeyCard)."
+            Label="Encrypted Backup Key"
+            name="backupKey"
+            rows={4}
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextarea
+            HelperText="Bitgo Public key (box C of the KeyCard)."
+            Label="BitGo Public Key"
+            name="bitgoKey"
+            rows={4}
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="The base address of the wallet (also known as root address)"
+            Label="Base Address"
+            name="rootAddress"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikPasswordfield
+            HelperText="Your wallet passphrase."
+            Label="Wallet Passphrase"
+            name="walletPassphrase"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="The recipient address for the recovery transaction. If a memo ID is required, append it to the address. For example: 0.0.1234?memoId=3"
```

Comment:
Algo uses consolidation
```suggestion
            HelperText="The recipient address for the recovery transaction."
```